<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f6f6f2
</string>
  </property>
  <widget class="QPushButton" name="placeOrder_btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(194, 237, 206);
color: rgb(73, 25, 1);
border-radius:15px
}
QPushButton:hover
{
background-color:#F8C471;
}</string>
   </property>
   <property name="text">
    <string>Place Order</string>
   </property>
  </widget>
  <widget class="QLabel" name="shopcart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:8px;
background-color: rgb(194, 237, 206);
border-radius:8px;
color: #717171;
	color: rgb(73, 25, 1);
border: 1px solid #CACFD2;
shopcart
{
box-shadow: 0 1px 6px 0 #d93025;
}</string>
   </property>
   <property name="text">
    <string>Shopping Cart</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #badfe7;
background-color: #DFFF00;
border-radius:10px;</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>338</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Delivery Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="place_order_response">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="delivery_addr_line_edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
